src/LiveComponent/src/Metadata/LivePropMetadata.php

```diff
@@ -53,6 +54,14 @@ public function allowsNull(): bool
         return $this->allowsNull;
     }
 
+    /**
+     * @return array{'name': string}
```

Comment:
`array{'name'?: string} `

or 

`array{'name': string}|null `

I'd rather have a nullable here cause at least if this is not null we will be certain to have a "name" key.